docs/compliance/cis/docker_ce.md

```diff
@@ -0,0 +1,12 @@
+---
+description: CIS Docker CE Benchmark information
+keywords: standards, compliance, security, benchmark
+title: CIS Docker CE benchmark
+---
+
+The Center for Internet Security (CIS) Docker Community Edition (CE) Benchmark is a reference document that can be used by system administrators, security and audit professionals and other IT roles in order to establish a secure configuration baseline for the Docker CE Engine. CIS Benchmarks are developed by an open community of security practitioners and licensed under a [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 Internal Public License](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode).
```

Comment:
...reference document designed to assist system administrators, security and audit professionals, and other technologists in establishing a secure...

Reply:
Will update